Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private string[] _amounts;
+
+        private void SetBalance(string[] amounts)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'SetBalance'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)